plugins/plugin-bash-like/src/preload.ts

```diff
@@ -23,14 +23,68 @@ import { CommandRegistrar } from '@kui-shell/core/models/command'
 
 import { preload as registerCatchAll } from './lib/cmds/catchall'
 
+import { getLoginShell, setShellAliases } from './pty/server'
+import { exec } from 'child_process'
+import * as propertiesParser from 'properties-parser'
+function prefetchEnv() {
```

Comment:
We could use `promisify` to make the code more readable?

```
const execAsync = promisify(exec)

const prefetchEnv = async () => {
  try {
    const data = await execAsync(`${await getLoginShell()} -l -c printenv`)
    if (data.stderr) {
      console.error(data.stderr)
    } else {
      const env = propertiesParser.parse(data.stdout.toString())
      debug('got env', env)
      for (const key in env) {
        process.env[key] = env[key]
      }
    }
  } catch (err) {
    console.error('Error retrieving shell ENV', err)
  }
}
```

This way we are more consistent in the decision to use async-await.